commands/base.go

```diff
@@ -144,6 +144,21 @@ func ParseCustomLabels(labels string) (map[string]string, error) {
 	return result, nil
 }
 
+func GetDisableCollectors(collectors string) []string {
```

Comment:
maybe it's better to call this method like a method above `ParseCollectorsList` or just `ParseList`?

Reply:
Renamed to ParceDisableCollectors for convenience. But as we do not can/need validate disabled metrics I do not see a reason to call it as the method above.